web/src/main/webapp/js/graphhopper/GHRequest.js

```diff
@@ -159,7 +163,7 @@ GHRequest.prototype.createGPXURL = function (withRoute, withTrack, withWayPoints
 };
 
 GHRequest.prototype.createHistoryURL = function () {
-    return this.createPath("?" + this.createPointParams(true));
+    return this.createPath("?" + this.createPointParams(true)) + "&useMiles=" + !!this.useMiles;
```

Comment:
createPath is also used by, e.g., createGPXURL etc. Because use_miles is nothing the Java API should worry about I thought it should not be put into createPath. Of course, having use_miles inside GHRequest is a bit of a workaround, because I assume GHRequest was meant to be the request object that communicates with graph hopper, use_miles (and do_zoom (?)) are more about the client side representation of the results.